run_tournament.py

```diff
@@ -41,14 +51,27 @@ def run_tournament(turns, repetitions, exclude_basic, exclude_strategies, exclud
             with open(fname, 'w') as f:
                 f.write(csv)
 
+            if not matplotlib_installed:
+                continue
+
             boxplot = axelrod.BoxPlot(results)
-            if boxplot.matplotlib_installed:
-                figure = boxplot.figure()
-                figure.savefig(plot, bbox_inches='tight')
-                figure.clf()
-            else:
-                print ("The matplotlib library is not installed. "
-                       "Only .csv output will be produced.")
+            figure = boxplot.figure()
+            figure.savefig(plot, bbox_inches='tight')
+            figure.clf()
+            
+            # Save plot with average payoff matrix with winners at top.
+            pmatrix_ranked = [[results.payoff_matrix[r1][r2] for r2 in results.ranking] for r1 in results.ranking]
+            fig, ax = plt.subplots()
```

Comment:
Does this not need to be in the line `if boxplot.matplotlib_installed:` statement?